README.md

```diff
@@ -6,21 +6,21 @@
 [![Security](https://github.com/jdrhyne/nutrient-dws-typescript-client/actions/workflows/security.yml/badge.svg)](https://github.com/jdrhyne/nutrient-dws-typescript-client/actions/workflows/security.yml)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-A TypeScript client library for [Nutrient Document Web Services (DWS) API](https://nutrient.io/). This library provides an isomorphic, type-safe, and ergonomic interface for document processing operations including conversion, merging, compression, watermarking, and text extraction.
+A Node.js TypeScript client library for [Nutrient Document Web Services (DWS) API](https://nutrient.io/). This library provides a type-safe and ergonomic interface for document processing operations including conversion, merging, compression, watermarking, and text extraction.
 
-> **Note**: This package is published as `nutrient-dws-typescript-client` on NPM. The package provides full TypeScript support and works in both Node.js and browser environments.
+> **Note**: This package is published as `nutrient-dws-typescript-client` on NPM. The package provides full TypeScript support and is designed specifically for Node.js environments.
 
 ## Features
 
-- 🌐 **Isomorphic**: Works in both Node.js and browser environments
+- 📄 **Powerful document processing**: Convert, OCR, edit, compress, watermark, redact, and digitally sign documents
+- 🤖 **LLM friendly**: Built-in support for popular Coding Agents (Claude Code, GitHub Copilot, Cursor, Windsurf) and documentation on Context7
+- 🔄 **100% mapping with DWS API**: Complete coverage of all Nutrient DWS API capabilities
```

Comment:
technically it's the DWS Processor API, not just the DWS API which also includes the Viewer API